### DISCUSSION TITLE
Second Assignment Project Help!!

### DISCUSSION THREAD

**Discussion by gauravchakrabarty** [cu-esiil-edu/2024-data-short-course#106] [Q&A]
@nquarder @eculler 
Hi Sir/Mam
I would appreciate your guidance on the second project. 
I thought of some Land Use/Land Cover classification (supervised / unsupervised) applying vegetation and water indices and extracting the areas in square km. 
But I have no idea how to do it using code space.

**Comment by nquarder** [comment]
Hi @gauravchakrabarty - Have you tried creating a repository for this project? You can try adding the data to the repo as long as it's not too large. Then you can use the `.devcontainer` file that we were using for the vegetation assignment so that you will have a working environment. From there you can try Creating a Codespace on Main (green button) and running your analysis. If there are libraries that you need to do the classification that are not part of our working environment, you will need to add them using a `!pip install` statement. 

Give this a try to see if you can get going, sir! -NQ

**Reply by gauravchakrabarty** [reply]
@nquarder 
Sure Sir!!
